pkg/pillar/cmd/zedkube/aitoapiserver.go

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build kubevirt
+
+package zedkube
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/lf-edge/eve/pkg/pillar/base"
+	"github.com/lf-edge/eve/pkg/pillar/kubeapi"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/sirupsen/logrus"
+	"github.com/vishvananda/netlink"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// checkIoAdapterEthernet - check and create NAD for direct-attached ethernet
+func checkIoAdapterEthernet(ctx *zedkubeContext, aiConfig *types.AppInstanceConfig) error {
+
+	if aiConfig.FixedResources.VirtualizationMode != types.LEGACY {
```

Comment:
Can you change this to types.NOHYPER
In PR https://github.com/lf-edge/eve/pull/3833 I was using NOHYPER.

Reply:
Sure. Updated.